test/integration_tests/abao/abao_test_hooks.js

```diff
@@ -83,6 +83,7 @@ hooks.skip("POST /projects/{ProjectId}/recalc -> 200")
 
 // Porting to python as per #600
 hooks.skip("POST /jobs/add -> 200")
+hooks.skip("PUT /jobs/{JobId} -> 200")
```

Comment:
- Added python test for jobs/{JobId}/cancel
- There's no delete for jobs
- Abao uses the first job from GET /jobs
- Cannot modify canceled job

It was quicker to skip it (it's already covered in python) than to raw delete it and/or getting Abao to use a different job.